### ISSUE TITLE
Up for adoption?

### ISSUE THREAD

**Issue by Alonski** [jasonkriss/ember-scroll-to#45]
When an original owner no longer has time to maintain the addon, there is a place where the addon can be "put up for adoption." Melanie Sumner has created [adopted-ember-addons](https://github.com/adopted-ember-addons).

Would you consider allowing us to adopt ember-scroll-to so that the addon may continue to provide useful service to the community?

**Comment by blimmer** [comment]
We ultimately have to find a way to contact @ragnarpeterson . He's the only one who has admin on the repository and can publish to npm.

**Comment by Alonski** [comment]
Maybe we could depreciate this and maintain a fork with a different name? Ember Observer might be able to get this one unlisted

**Comment by alanlalonde** [comment]
NPM has a disputes process that can facilitate transfer of rights on that side of things. All published packages need some sort of contact information in order to reach the owner.

https://www.npmjs.com/policies/disputes

**Comment by blimmer** [comment]
I sent an email today in accordance with the NPM policy disputes linked above. If the owner doesn't respond in 4 weeks, we should be able to take over ownership on the NPM side.

![screen shot 2018-11-02 at 10 49 57](https://user-images.githubusercontent.com/630449/47928923-33771f80-de8d-11e8-8e92-b30c9877d44f.png)

**Comment by Alonski** [comment]
Awesome work @blimmer Hope we can get this addon running smoothly again :)

**Comment by blimmer** [comment]
I now have publish rights on this package and have published 0.6.5.

That said, **I don't have bandwidth to maintain this project**. Please reach out to me on the Ember Discord (@blimmer) if you're interested in taking ownership of this package.

cc @Alonski @Duder-onomy @alanlalonde

**Comment by RobbieTheWagner** [comment]
@blimmer can we transfer this to the adopted-ember-addons org?

**Comment by blimmer** [comment]
@rwwagner90 absolutely! However, I don't have repo admin so you'd need to get in touch with GitHub or fork it.

**Comment by RobbieTheWagner** [comment]
Does anyone know how to reach @ragnarpeterson?

**Comment by Alonski** [comment]
@rwwagner90 Do you want to open a fork of this? Would you be up for maintaining it?

**Comment by RobbieTheWagner** [comment]
@Alonski I already maintain quite a lot of repos, but I could occasionally help out with maintenance.

**Comment by Alonski** [comment]
@rwwagner90  I personally don't use this so I can't maintain it :)

**Comment by RobbieTheWagner** [comment]
I don't use it either, but occasionally I see it used in apps that I end up working on.
